x/pylons/types/payment_processor_test.go

```diff
@@ -148,6 +190,26 @@ func TestPaymentProcessor_ValidateRedeemInfo(t *testing.T) {
 			},
 			wantErr: false,
 		},
+		{
+			name: "InvalidPubkey",
+			fields: fields{
+				CoinDenom:            "ustripeusd",
+				PubKey:               "INVALID_PUBKEY",
+				ProcessorPercentange: DefaultProcessorPercentage,
```

Comment:
ProcessorPercentage is correct spelling.

Reply:
Thank you very much! I have fix in commit https://github.com/Pylons-tech/pylons/pull/870/commits/8aa682a4948fce1f64f88c04f253cbb7e3b4e2f1